python/vyos/ifconfig/bridge.py

```diff
@@ -365,6 +365,21 @@ def update(self, config):
                     if native_vlan_id:
                         cmd = f'bridge vlan add dev {interface} vid {native_vlan_id} pvid untagged master'
                         self._cmd(cmd)
+            
+            # When the bridge port is initialized, set the standby port now,
+            # This reports an error if the bridge did not complete the member
+            # port addition before setting the standby port
+            for interface, interface_config in member_intf.items():
```

Comment:
Duplicated code in interface.py and bridge.py

Reply:
done